python/requirements/ml/requirements_tune.txt

```diff
@@ -1,11 +1,8 @@
 -r requirements_dl.txt
 
 aim==3.16.1
-ax-platform[mysql]==0.2.6
-# Newer version of gpytorch are incompatible with ax 0.2.6.
-# Todo: Remove pin when upgrading ax
-gpytorch==1.8.1
```

Comment:
Do we need to keep this for python 3.7 which uses 0.2.6?

Reply:
We don't - 1.8.1 is the latest available gpytorch version for Python 3.7 (see https://github.com/cornellius-gp/gpytorch/blob/fd99baf587f6f50f414bb53b39a08a2f7d7a5e5b/setup.py for 1.9.0), so this is implicitly pinned in Python 3.7.